docking.js

```diff
@@ -350,7 +350,7 @@ const DockedDash = new Lang.Class({
         ]);
 
         this._injectionsHandler = new Utils.InjectionsHandler();
-        this._themeManager = new Theming.ThemeManager(this._settings, this.actor, this.dash);
+        this._themeManager = new Theming.ThemeManager(this._settings, this.actor, this.dash, this);
```

Comment:
Could be just: 
```this._themeManager = new Theming.ThemeManager(this._settings, this);```

Reply:
Thanks. Included this here [e0fc16a2f1dae821d2f91e1695dd27beea0512c0].